modules/wowza/main.tf

```diff
@@ -298,6 +301,12 @@ data "azurerm_key_vault_secret" "ssh_pub_key" {
   key_vault_id = data.azurerm_key_vault.cvp_kv.id
 }
 
+data "azurerm_key_vault_secret" "dynatrace_token" {
```

Comment:
I would add the token in via TF as well. Put it in the ADO Library and pull it into TF to add into the vault or just straight into the script